rpc/gnmi/gnmi-specification.md

```diff
@@ -1603,10 +1605,10 @@ affecting the sample accuracy and freshness to the client, and as a result, on
 the client's ability to react to events on the target.
 
 Since it is not possible for the target to infer whether its clients are
-sensitive to the latency introduced by bundling, if a target implements
-optimizations such that multiple `Update` messages are bundled together,
-it MUST provide an ability to disable this functionality within the
-configuration of the gNMI service. Additionally, a target SHOULD provide means
+sensitive to the latency introduced by bundling, the target MUST follow the
+`SubscriptionList` fields in [Section 3.5.1.2](#3512-the-subscriptionlist-message)
+that indicate if `allow_aggregation` is permitted.
```

Comment:
I think that maybe we need a larger change to the wording here. I think we should clarify what _bundling_ means here vs. _aggregation_. Specifically, we were trying to give guidance here as to how many `update` messages there were per `notification`.

In aggregation, I think we are looking at how many leaves per `update` there are. i.e., without `allow_aggregation` this should be 1. 

I'd propose we make this clarification, and then add some text as to when aggregation is triggered. WDYT?